src/components/pages/TeachersPage/Conversation.tsx

```diff
@@ -6,13 +6,14 @@ import { Box, Typography } from '@mui/material';
 import conversationCSS from './Conversation.css';
 import { filterWords, scrollDown } from '@utils/classrooms';
 
-export default function Conversation({ chat }) {
+export default function Conversation({ chat, inAllStudentsChatDisplay }) {
   const [student1, student2] = chat.studentPair;
   const lastMessage = useRef(null);
 
   useEffect(() => {
-    if (chat.chatId !== 'homepage sample chat') scrollDown(lastMessage);
-  }, [chat.chatId, chat.conversation]);
+    if (!inAllStudentsChatDisplay && chat.chatId !== 'homepage sample chat')
```

Comment:
@mssiegel right that makes sense. Regarding the dependency array, would we want to keep the `chat.conversation` so that it scrolls on every new message? Or else it would only scroll when we switch the displayed chat.

Reply:
@karlrez thats right. Good catch!